rules/detekt/src/main/kotlin/com/twitter/compose/rules/detekt/ComposeModifierWithoutDefaultCheck.kt

```diff
@@ -14,8 +14,6 @@ class ComposeModifierWithoutDefaultCheck(config: Config) :
     TwitterDetektRule(config),
     ComposeKtVisitor by ComposeModifierWithoutDefault() {
 
-    override val autoCorrect: Boolean = true
```

Comment:
Why this removed?

Reply:
It was a bug that it was there, it causes a crash with detekt when triggered (e.g. from the cli).